modules/installation-aws-delete-cluster.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+:_content-type: PROCEDURE
+[id="installation-aws-delete-cluster"]
+= Deleting a cluster with a configured AWS Local Zone infrastructure
+
+After you install a cluster on Amazon Web Services (AWS) into an existing Virtual Private Cloud (VPC), and you set subnets for each Local Zone location, you can delete the cluster and any AWS resources associated with it. 
+
+The example in the procedure assumes that you created a VPC and its subnets by using a CloudFormation template.
+
+.Prerequisites
+
+* You can access to the copied CloudFormation template and the stack of AWS resources that you created with the template.
+* You can access the installation configuration file that can deploy your cluster.
+* Optional: If you configured SSH access to your cluster, ensure you know the public and private key pair.
+* You logged in to the AWS CloudFormation CLI console.
```

Comment:
Need to review, for each bullet:

- The user don't need the CloudFormation template, **only the stack name**
- The user don't need the IC, he needs to know the install-dir that the `openshift-install` will point with arg `--dir`
- Dont need SSH in this section
- Need to review that statement. Dont need to access the **AWS Console**, only valid credentials to delete the cloudformation stack. Please take a look to the UPI with CloudFormation as reference: https://docs.openshift.com/container-platform/4.12/installing/installing_aws/installing-aws-user-infra.html#installation-aws-user-infra-delete-bootstrap_installing-aws-user-infra

Reply:
Thanks, @mtulio . I was off the mark with these prereqs. I updated the list; please let me know your thoughts.